MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -312,6 +312,7 @@ def main():
     parser.add_argument("-tf", type=int, help="number of timeframes per job", default=1)
     parser.add_argument("--ccdb-IRate", type=bool, help="whether to try fetching IRate from CCDB/CTP", default=True)
     parser.add_argument("--trig-eff", type=float, dest="trig_eff", help="Trigger eff needed for IR", default=-1.0)
+    parser.add_argument("-enable-parallel-world", type=int, dest="enable_parallel_world", help="Enable parallel geometry", default=0)
```

Comment:
This should not be needed. We can already pass simulation config key values from the outside.

Reply:
Thanks for your review, I removed this in the latest commit